Source/URLDispatcher.spoon/init.lua

```diff
@@ -28,6 +28,22 @@ obj.default_handler = "com.apple.Safari"
 --- If true, handle Slack-redir URLs to apply the rule on the destination URL. Defaults to `true`
 obj.decode_slack_redir_urls = true
 
+--- URLDispatcher.url_redir_decoders
+--- Variable
+--- List containing optional additional redirection decoders (other
+--- than the known Slack decoder, which is enabled by
+--- `URLDispatcher.decode_slack_redir_urls` to apply to URLs before
+--- dispatching them. Each list element must be a list itself with four
+--- elements:
+---   * a name to identify the decoder
+---   * a regex to match against the URL
```

Comment:
I think `regex` is the wrong thing to say here - `string.gsub()` uses normal Lua string patterns, right? They're not quite the same as regexes.

Reply:
Duh! Of course, fixed. Habits... 😆